docs/community/working-group-definitions.md

```diff
@@ -0,0 +1,198 @@
+* @anmenaga
+
+## Developer Experience
+
+The PowerShell developer experience includes the **development of modules** (in C#, PowerShell script, etc.),
+as well as the experience of **hosting PowerShell and its APIs** in other applications and language runtimes.
+Special consideration should be given to topics like **backwards compatibility** with Windows PowerShell
+(e.g. with **PowerShell Standard**) and **integration with related developer tools**
+(e.g. .NET CLI or the PowerShell extension for VS Code).
+
+### Members
+
+* @JamesWTruher (PS Standard, module authoring)
+* @adityapatwardhan (SDK)
+* @rjmholt (hosting, WinPS compatibility)
+
+## Engine
+
+The PowerShell engine is one of the largest and least well-defined aspects of the codebase.
+The Engine WG should be focused on the
+**implementation and maintenance of core PowerShell engine code**.
+This includes (but is not limited to):
+
+* The language parser
+* The command and parameter binders
+* The module and provider systems
+    * `*-Item` cmdlets
+    * Providers
+* Performance
+* Componentization
+* AssemblyLoadContext
+
+It's worth noting that the Engine WG is not responsible for the definition of the PowerShell language.
+This should be handled by the Language WG instead.
+However, it's expected that many issues will require input from both WGs.
+
+### Members
+
+* @daxian-dbw
+* @rjmholt
+* @BrucePay
+
+## Interactive UX
+
+While much of PowerShell can be used through both interactive and non-interactive means,
+some of the PowerShell user experience is exclusively interactive.
+These topics include (but are not limited to):
+
+* Console
+* Help System
+* Tab completion / IntelliSense
+* Markdown rendering
+* PSReadLine
+* Debugging
+
+### Members
+
+* @daxian-dbw (PSReadline / Intellisense)
+* @adityapatwardhan (Markdown / help system)
+* @JamesWTruher (cmdlet design)
+
+## Language
+
+The Language WG is distinct from the Engine WG in that they deal with the abstract definition
+of the PowerShell language itself.
+While all WGs will be working closely with the PowerShell Committee (and may share members),
+it's likely that the Language WG will work especially close with them,
+particularly given the long-lasting effects of language decisions.
+
+### Members
+
+* @JamesWTruher
+* @rjmholt
+* @daxian-dbw
+* @BrucePay
+
+## Remoting
+
+The Remoting WG should focus on topics like the **PowerShell Remoting Protocol (PSRP)**,
+the **protocols implemented under PSRP** (e.g. WinRM and SSH),
+and **other protocols used for remoting** (e.g. "pure SSH" as opposed to SSH over PSRP).
+Given the commonality of serialization boundaries, the Remoting WG should also focus on
+**the PowerShell job system**.
+
+### Members
+
+* @PaulHigin
+* @SteveL-MSFT
+* @TravisEz13
+* @anmenaga
+
+## Modules
+
+For those core/inbox modules whose source code lives within the `PowerShell/PowerShell` repository,
+we should have some set of WGs responsible for their cmdlet design guidelines and enforcement,
```

Comment:
Just calling out that this is definitely WIP (as expected given this PR is WIP): what are the set of WGs that we're setting up for the modules? We should either define that now, or at least setup one "Modules" WG. This section feels like it's doing just that, but it's not explicit about it.